client/components/wordpress-logo/docs/example.jsx

```diff
@@ -0,0 +1,31 @@
+/**
+* External dependencies
+*/
+import React from 'react';
+
+/**
+ * Internal dependencies
+ */
+import WordPressLogo from 'components/wordpress-logo';
+import Card from 'components/card';
+
+module.exports = React.createClass( {
+	displayName: 'WordPressLogo',
+
+	render() {
+		return (
+			<div className="design-assets__group">
+				<h2>
+					<a href="/devdocs/design/wordpress-logo">WordPress Logo</a>
+				</h2>
+				<Card>
```

Comment:
> @jasmussen do you remember what should we encourage here?

For Gridicons, the best size is 24 and 48. 36 is also good. 18 is okay to use in edgecases.

The unique situation with the WordPress logo was that the one bundled in Gridicons, is less the WordPress logo, and more a gridicon specifically balanced to work with the other mastebar icons, Reader, Notifications, Me. That means the Gridicon for WordPress has a 2pt stroke in the Gridicon.

So it all depends on where these logos are to be used. If they are to be used as *logos*, I think the answer is that many sizes are okay all depending on context, so we might as well pick one. However specifically with regards to the Jetpack icon when used very small, I think there's a pixel hinted version. @drw158 will know more.